libraries/AP_NavEKF2/AP_NavEKF2_Measurements.cpp

```diff
@@ -480,6 +501,7 @@ void NavEKF2_core::readGpsData()
             }
 
             // convert GPS measurements to local NED and save to buffer to be fused later if we have a valid origin
+            // correct for position offset of antenna in body frame
```

Comment:
There's a comment change here that implies a functional code change, but no code change.